sunpy/net/helioviewer.py

```diff
@@ -22,44 +22,75 @@
 
 
 class HelioviewerClient(object):
-    """Helioviewer.org Client"""
-    def __init__(self, url="https://legacy.helioviewer.org/api/"):
+    """Helioviewer.org Client"""      
```

Comment:
Could you remove the extra white space at the end?

Reply:
Ok